utils/index.d.ts

```diff
@@ -1430,6 +1430,11 @@ export declare abstract class AzureWizardExecuteStep<T extends IActionContext &
      */
     public abstract shouldExecute(wizardContext: T): boolean;
 
+    /**
+     * Optionally returns AzureWizardExecuteSteps that will be injected and sorted into the Azure Wizard executeSteps array by priority
```

Comment:
This behavior could result in steps happening out of the expected order, e.g. if I inject one with priority 90, from a step with priority 100, it'll happen in reverse order of usual. We should maybe note that in the method description.

Reply:
I can note that, but if someone injects a step with a higher priority than the one that created it, I think they deserve to have some bugs.

But I do re-sort the array by priority every time a step is injected.